logstash-core/lib/logstash/settings.rb

```diff
@@ -176,18 +176,43 @@ def validate(value)
       end
     end
 
-    ### Specific settings #####
-
-    class Boolean < Setting
-      def initialize(name, default, strict=true, &validator_proc)
+    class Coercible < Setting
```

Comment:
Does this means we can have settings that could be `Coercible` and some that not?